src/main/content/blog.html

```diff
@@ -57,20 +57,20 @@ <h2 class="blog_subtitle">
      {% t blog.featured_tags %}
     </h2>
     <div id="featured_tags_list" role="list">
-     <p class="featured_tag" data-tag-id="announcements" role="button" tabindex="0">
```

Comment:
I think these were recently changed to button in prod: https://github.com/OpenLiberty/openliberty.io/blob/prod/src/main/content/blog.html#L60